### PR TITLE
Change text of About page

### PR DESCRIPTION
**Pull request by andsemenov**

### CODE REVIEW

File: client/src/components/About.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return <div>A diversity-focused recruitment platform.</div>;
```
Comment (llh1) [inline client/src/components/About.js]: I think we need to have the "About" title in the page and then a description of the project.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/About.js (hunk 1 above)
Comment (gregdyke) [reply client/src/components/About.js]: Maybe we can delay that until the team + PM have decided what actually goes in the about page? (more generally, I think we should consider approving changes unless they go against the acceptance criteria of the ticket)

**Review by DwinaTech — APPROVED** [review]
Good work 👍🏿

**Review by llh1 — APPROVED** [review]
Looks good :)

**Pull request merged**